### DISCUSSION TITLE
Use custom emoji on another sever

### DISCUSSION THREAD

**Discussion by Fusity** [Rapptz/discord.py#6757] [General]
It is possible to use custom emojis from my server to an another server ?
Because when i'm using the `<:emoji_name:emoji_id>` on an another serveur but i can't print it ;(

**Comment by AbstractUmbra** [comment]
A bot account can use emoji from any server it is a member of (aside from role gated ones iirc).

The format you have is correct, it should be noted though that animated emoji start with `<a:` and not `<:`.

Can you provide some actual code to show this not working, and maybe what is actually output?

**Reply by DerekJarvis** [reply]
THANK YOU! the `a` is what I was missing to make this work.

**Comment by Fusity** [comment]
Of course !
`@bot.event
async def on_command_error(ctx, error):
    if isinstance(error, commands.MissingRequiredArgument):
        await ctx.send("<:PacifiqueCreation22:#676896410184056856#> Il manque un argument. - <:PacifiqueCreation22:#676896410184056856#>")`
        
Yeh i'm french !
And here the output :
       :PacifiqueCreation22: Il manque un argument. - :PacifiqueCreation22:
       
       I don't like github post script :( or i don't know how to use it !

**Reply by mesub7** [reply]
Try it without the `#`

**Reply by AbstractUmbra** [reply]
For more clarification:

Remove the `#` from the ID block of the custom emoji format:
`<:name:#ID>` becomes `<:name:ID>`

**Reply by Fusity** [reply]
This works in the server where the emoji is hosted but not in another server where the emoji is not hosted

**Reply by AbstractUmbra** [reply]
Your bot has to be a member of the server where the emoji is hosted. It cannot use emoji it cannot logically 'see'.

**Comment by Fusity** [comment]
@Rapptz you can close

**Reply by Rapptz** [reply]
I can't close discussions. If you feel like it was answered then you can press the "Mark as Answer" button on the answer you felt was most appropriate.

![image](https://user-images.githubusercontent.com/1695103/116336590-d41cd080-a7a6-11eb-8ba4-9db9ac337c01.png)
